modules/data-entry/src/main/frontend/src/dataHomepage/DeleteButton.jsx

```diff
@@ -78,6 +75,10 @@ function DeleteButton(props) {
       setErrorMessage(`${entryName} could not be found. This ${entryType ? entryType : "item"} may have already been deleted.`);
       setEntryNotFound(true);
       openError();
+    } else if (status === 401 || response.url.startsWith(window.location.origin + "/login")) {
+      setErrorMessage(`The currently signed in user does not have permission to delete ${entryName}.`);
```

Comment:
The message is technically correct, but a bit too technical. I would suggest something simpler instead:
```suggestion
      setErrorMessage(`You do not have permission to delete ${entryName}.`);
```